docs/sql-programming-guide.md

```diff
@@ -1024,6 +1024,7 @@ the Data Sources API. The following options are supported:
       The JDBC URL to connect to.
```

Comment:
Could you also improve this? You know, `url` might contain `user` and `password`, if users do not provide `user` and `password` in the option list. 

BTW, I think we also need to document `user` and `password` here.